central/compliance/service/service_impl.go

```diff
@@ -127,9 +127,19 @@ func (s *serviceImpl) GetAggregatedResults(ctx context.Context, request *v1.Comp
 		return nil, err
 	}
 
+	filteredSources := sources[:0]
+	for _, source := range sources {
+		config, exsist, errConfig := s.complianceDataStore.GetConfig(ctx, source.GetStandardId())
```

Comment:
typo: `exsist`

Reply:
@ksurabhi91 Looks like the typo still exists. errConfig should just be err